.github/workflows/deploy-docs.yml

```diff
@@ -27,9 +27,11 @@ jobs:
         with:
           persist-credentials: false
           fetch-depth: 0
+          path: "."
 
       - name: Build 🔧
         run: |
+          git config --global --add safe.directory /__w/cosmos-sdk/cosmos-sdk
```

Comment:
Weird, this was not required in the previous workflow, as `actions/checkout` should be adding it.
Anyhow, this fixes the docs build (see it live at https://docs.cosmos.network/main)